### PR TITLE
Kafka 1.1.0!

### PR DESCRIPTION
**Pull request by stevenschlansker**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -19,7 +19,7 @@
   <parent>
     <groupId>com.opentable</groupId>
     <artifactId>otj-parent-spring</artifactId>
-    <version>139</version>
+    <version>144-SNAPSHOT</version>
```
Comment (stevenschlansker) [inline pom.xml]: this will be updated to a non-snapshot before merging

**Comment by mikebell90** [comment]
If you could get otj-rest-template 1.0.7 PR in too, much obliged.

File: src/main/java/com/opentable/kafka/embedded/EmbeddedKafkaBroker.java [hunk 2]
```diff
@@ -62,6 +66,7 @@
 
     private EmbeddedZookeeper ezk;
     private KafkaServer kafka;
+    private AdminClient admin;
```
Comment (pennello) [inline src/main/java/com/opentable/kafka/embedded/EmbeddedKafkaBroker.java]: Is there a reason we can't make this final, initializing it earlier?

File: src/main/java/com/opentable/kafka/embedded/EmbeddedKafkaBroker.java [hunk 3]
```diff
@@ -108,14 +116,45 @@ public EmbeddedKafkaBroker start()
         return this;
     }
 
+    // Create consumer offsets ourselves so we can control replication, partitions, etc
+    private void maybeCreateConsumerOffsets() throws InterruptedException {
```
Comment (pennello) [inline src/main/java/com/opentable/kafka/embedded/EmbeddedKafkaBroker.java]: Why is this named like it is?  Seems like it's just polling on describe topics.

Comment (mikebell90) [reply src/main/java/com/opentable/kafka/embedded/EmbeddedKafkaBroker.java]: Yeah I think @stevenschlansker pulled this from chat, where there is a condition. The name doesn't make a lot of sense now.

File: src/main/java/com/opentable/kafka/embedded/EmbeddedKafkaBroker.java [hunk 4]
```diff
@@ -108,14 +116,45 @@ public EmbeddedKafkaBroker start()
...
+        final String consumerOffsets = "__consumer_offsets";
+        Map<String, TopicDescription> description;
+        final Instant start = Instant.now();
+        LOG.info("Creating consumer offsets");
+        while (true) {
+            try {
+                description = admin.describeTopics(Collections.singleton(consumerOffsets)).all().get(10, TimeUnit.SECONDS);
+                if (description.isEmpty()) {
+                    createTopic(consumerOffsets);
```
Comment (stevenschlansker) [inline src/main/java/com/opentable/kafka/embedded/EmbeddedKafkaBroker.java]: note the create right here

**Pull request merged**